apps/frontend/app/user/agents/creation/upload-form.tsx

```diff
@@ -196,31 +210,33 @@ function UploadForm({
                           {...{ onBlur, disabled, name, ref }}
                         />
                       </FormControl>
-                      <FormDescription>
+                      <FormDescription className="text-gray-300">
                         Upload a character JSON file.
                       </FormDescription>
-                      <FormMessage />
```

Comment:
Put `<FormMessage />` back in; users need to be able to see the feedback when they make an error.